eng/containers/UbuntuNetCoreKeyring/Dockerfile

```diff
@@ -1,10 +1,10 @@
-FROM mcr.microsoft.com/dotnet/core/sdk:3.1-bionic AS build
+FROM mcr.microsoft.com/dotnet/sdk:5.0-focal AS build
 
 ENV \
     NO_AT_BRIDGE=1 \
     DOCKER_CONTAINER_NAME="ubuntu_netcore_keyring" \
     PSModuleAnalysisCachePath=/var/cache/microsoft/powershell/PSModuleAnalysisCache/ModuleAnalysisCache \
-    POWERSHELL_DISTRIBUTION_CHANNEL=PSDocker-DotnetCoreSDK-Ubuntu-18.04
+    POWERSHELL_DISTRIBUTION_CHANNEL=PSDocker-DotnetCoreSDK-Ubuntu-20.04
```

Comment:
Does .NET 5 require Ubuntu20?

Reply:
I mostly ask because we are still using ubuntu 18 in most of our other pipelines.

Reply:
I didn't see a docker image for 5.0 based on Ubuntu 18 in thee the full tag listing [here](https://hub.docker.com/_/microsoft-dotnet-sdk). If you think it's an issue to use Ubuntu 20 here we can build our own image.

Reply:
Interesting they seem to support .NET 5 on 18.04 (https://docs.microsoft.com/en-us/dotnet/core/install/linux#ubuntu) but they don't publish a docker container. I don't know of any specific issues with Ubuntu 20.04 currently but I know the last time we tried moving there were some sorting issues due to some cultural library changes in the OS. You can certainly try 20.04 if that is easier but if you start seeing weird issues we might want to reconsider until we move everything to 20.04.

Reply:
@weshaggard the only issue we've found so far was python was not installed by default, but if we see more we can build one from 18.04.